shared/coordinator/src/coordinator.rs

```diff
@@ -251,6 +250,7 @@ pub struct CoordinatorConfig {
     pub global_batch_size_end: u16,
 
     pub verification_percent: u8,
+    pub waiting_for_members_extra_time: u8,
```

Comment:
just a nit but i'm a bit worried this introduces a breaking change, could you check if the new test added by this PR still passes?

https://github.com/PsycheFoundation/psyche/pull/124